mcap_vendor/CMakeLists.txt

```diff
@@ -68,13 +68,6 @@ macro(build_mcap_vendor)
   )
 
   install(TARGETS mcap EXPORT export_mcap)
-
-  install(
-    EXPORT export_mcap
-    FILE mcap-config.cmake
-    DESTINATION share/cmake/mcap/
-    NAMESPACE "mcap_vendor::"
-    EXPORT_LINK_INTERFACE_LIBRARIES)
```

Comment:
Why remove this? Do we need to keep `install(TARGETS mcap EXPORT export_mcap)`?

Reply:
> Why remove this?

Becaause `ament_export_targets` do the similar thing. https://github.com/ament/ament_cmake/blob/2bf27ce5ad01af340e1f4efd44232c1067d0dbda/ament_cmake_export_targets/cmake/ament_cmake_export_targets_package_hook.cmake#L28-L33

>  Do we need to keep install(TARGETS mcap EXPORT export_mcap)?

Yes, I think we need it.